.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: objective-c
+osx_image: xcode7.2
+xcode_project: MapPointMapper.xcodeproj
+xcode_scheme: MapPointMapper
+script: xctool -project MapPointMapper.xcodeproj -scheme MapPointMapper build CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO
```

Comment:
whoops,  yeah that should be `test`.  Awesome, slather and coveralls would be good one to add, but let's do that in a later PR